### ISSUE TITLE
We should be able to see everyone in the list individually in the details on group sms.

### ISSUE THREAD

**Issue by yusufsunetci** [FossifyOrg/Messages#147]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

When we send group sms, we should be able to see everyone in the list individually in the details. Both sent and forwarded. And it only shows phone numbers. We don't know all the numbers in the manual by heart. That's why the names should also appear in the details.

### Why do you want this feature?

I want to know who the message was delivered to.

**Comment by Aga-C** [comment]
What do you exactly mean, what's your single feature request here? That in details you see only number? Or that you want to see if everyone received a message?

**Comment by yusufsunetci** [comment]
In details, I see only number.  The names should also appear in the details.

**Comment by Aga-C** [comment]
Do you store contacts in `Phone storage (not visible by other apps)`?

**Comment by yusufsunetci** [comment]
Yes, my contacts in my phone.

**Comment by Aga-C** [comment]
In Fossify Contacts, do you have enabled `Show private contacts to Fossify Phone, Fossify SMS Messenger and Fossify Calendar` in Settings?

**Comment by yusufsunetci** [comment]
Yes I activated the program when I installed it. I've been using these apps since they were published on F-droid.

**Comment by Aga-C** [comment]
@yusufsunetci So it seems to be a bug, because those names should be shown (they are visible for contacts in other stores). Could you change the title to: `Private contact names are not visible in group message details`, so it'll describe better the problem for others to find it? Thanks!

**Comment by yusufsunetci** [comment]
> they are visible for contacts in other stores

Some people are registered in my contacts in the phone memory, some people are registered by mail account... None of them are visible. I've tried that too.
